erpnext/portal/product_configurator/utils.py

```diff
@@ -243,6 +244,8 @@ def get_next_attribute_and_values(item_code, selected_attributes):
 	else:
 		product_info = None
 
+	product_info["allow_items_not_in_stock"] = cint(frappe.db.get_singles_value("Shopping Cart Settings", "allow_items_not_in_stock"))
```

Comment:
From get_product_info_for_website you will get the info about the cart_settings
data.cart_settings.allow_items_not_in_stock?